testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -1162,6 +1162,38 @@ private void assertSelectsOnSpecialCharacters(String trinoTableName, String spar
         }
     }
 
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
+    public void testTrinoReadsSparkSortOrder()
+    {
+        String sourceTableNameBase = "test_insert_into_sorted_table_" + randomNameSuffix();
+        String trinoTableName = trinoTableName(sourceTableNameBase);
+        String sparkTableName = sparkTableName(sourceTableNameBase);
+
+        onSpark().executeQuery("CREATE TABLE " + sparkTableName + " (a INT, b INT, c INT) USING ICEBERG ");
+        onSpark().executeQuery("ALTER TABLE " + sparkTableName + " WRITE ORDERED BY b, c DESC NULLS LAST");
+
+        Assertions.assertThat((String) onTrino().executeQuery("SHOW CREATE TABLE " + trinoTableName).getOnlyValue())
+                .contains("sorted_by = ARRAY['b ASC NULLS FIRST','c DESC NULLS LAST']");
+
+        onTrino().executeQuery("INSERT INTO " + trinoTableName + " VALUES (1, 2, 3), (3, 2, 1), (NULL, NULL, NULL)");
+    }
```

Comment:
INSERT and no verification?
maybe skip the insert if you don't care about it results.

(or add a comment why it's a good idea to test that insert does not fail, but i think we're well covered by Iceberg BCT)

Reply:
I meant to come back and finish these tests but totally forgot...